src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css

```diff
@@ -425,77 +402,89 @@ div.table-tabs > button.table-tab {
    background: #4D7A97;
    color: #FFFFFF;
 }
-
-.row-color th,
-.alt-color th {
-    font-weight:normal;
-}
-.overview-summary td, .member-summary td, .type-summary td,
-.use-summary td, .constants-summary td, .deprecated-summary td,
-.requires-summary td, .packages-summary td, .provides-summary td,
-.uses-summary td, .system-properties-summary td {
+.two-column-summary {
+    display: grid;
+    grid-template-columns: minmax(15%, max-content) minmax(15%, auto);
+}
+.three-column-summary {
+    display: grid;
+    grid-template-columns: minmax(10%, max-content) minmax(15%, max-content) minmax(15%, auto);
+}
+.four-column-summary {
+    display: grid;
+    grid-template-columns: minmax(10%, max-content) minmax(10%, max-content) minmax(10%, max-content) minmax(10%, auto);
+}
```

Comment:
I like the minmax values and mostly like the effect on the generated docs ... although it is still slightly weird (to me) that the column widths can change when switching tabs in the same table.

Reply:
I agree. I did try to make column width more uniform, but it is nearly impossible to do that while fitting all possible use cases. Content lengths vary very much even within the same type of summary table.